### PR TITLE
Add to docs: comparison, operation, and allocate to N targets

### PR DESCRIPTION
**Pull request by tuwannu**

### CODE REVIEW

**Comment by tuwannu** [comment]
Travis's `make -C doc spelling` is reporting spelling errors as follow:

    features/comparison.rst:56:USD:
    features/comparison.rst:56:USD:
    features/comparison.rst:59:greaterThanOrEqual:

Not sure what is wrong. Can anyone help?

**Comment by sagikazarmark** [comment]
Add them to https://github.com/moneyphp/money/blob/master/doc/spelling_word_list.txt

**Comment by tuwannu** [comment]
@sagikazarmark Thanks. All green now.

The built docs is available for review at http://tuwannu-money.readthedocs.io/en/docs-new/

File: doc/features/allocation.rst [hunk 1]
```diff
@@ -18,3 +21,15 @@ is to keep handing out the remainder until all money is spent. In other words:
     // The order is important:
     list($investors_cut, $my_cut) = $profit->allocate(30, 70);
     // $my_cut is 3 cents, $investors_cut is 2 cents
+
+
+Allocate to N targets
+------------------------
+
+An amount of money can be allocated amount N targets using ``allocateTo()``.
```
Comment (sagikazarmark) [inline doc/features/allocation.rst]: allocated to N

File: doc/features/operation.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
+Operation
+=========
+
+.. ATTENTION:: All operations with Money objects are always immutable. See :ref:`Immutability`.
```
Comment (sagikazarmark) [inline doc/features/operation.rst]: All and always does not sound good IMO. Let's drop "All".

**Comment by sagikazarmark** [comment]
I like it, thanks @tuwannu. @frederikbosch might also want to review it though.

File: doc/features/operation.rst (hunk 2 above)
Comment (tuwannu) [inline doc/features/operation.rst]: My bad, typo error. Fixed.

**Comment by frederikbosch** [comment]
@tuwannu Great PR. We were indeed lacking comparison and operation docs.

**Comment by frederikbosch** [comment]
@tuwannu Please use the TOC from #313. That looks great.

**Comment by sagikazarmark** [comment]
I am merging this, #313 might go into a separate PR.

**Comment by sagikazarmark** [comment]
Thanks @tuwannu

**Pull request merged**